### ISSUE TITLE
Are there training data for phase2 and phase3?

### ISSUE THREAD

**Issue by Czwbna** [CellularLint/cellularlint-codes#1]
Thanks for your work first. I trained the models using snli_1.0_train.csv and phase1-mini.csv you provided, and ran eval.py to get the output_metrics.txt, following the instructions of Readme. The results are:
---------------Results from Model Bert---------------

Precision: 0.28723404255319146
Recall: 0.35000000000000003
Accuracy: 0.38181818181818183
F1 Score: 0.26439232409381663

---------------Results from Model Roberta---------------

Precision: 0.32007575757575757
Recall: 0.3499999999999999
Accuracy: 0.36363636363636365
F1 Score: 0.3189102564102564

---------------Results from Model XLNet---------------

Precision: 0.0909090909090909
Recall: 0.3333333333333333
Accuracy: 0.2727272727272727
F1 Score: 0.14285714285714285

---------------Results from Model Encell---------------

Precision: 0.36283891547049435
Recall: 0.3166666666666667
Accuracy: 0.32727272727272727
F1 Score: 0.2917771883289125

The scores of phase1 are not good as described in the paper.  And I don't know how to do training of phase2 and phase3 because I don't find training data for these 2 phases. I then tried to train the models using phase1-mini.csv for 2 more times, and the results were still not good.

I notice that in Section 4.1 1E, you explain the multi-phase informed fine-tuning: 
> we sample a small subset of data from the predicted set, then validate and rectify the labels through domain-expert human annotators. This approach ensures ground truth addition while keeping the human involvement minimal. Also, rectifying annotations instead of annotating from scratch reduces the human effort even more. The models are again trained on the reconditioned examples in a supervised manner and used subsequently for predictions, again on the whole dataset. In such a manner, we complete one phase of training. This method is followed k times (each defining a phase) before finalizing the prediction.

So, should I sample 2 more small subsets of data from the predicted set, then validate and rectify the labels manually for training?

**Comment by CellularLint** [comment]
Hello @Czwbna,

Thank you for your interest in our work. The current version of our code and dataset are part of the artifact evaluation at USENIX Security '24. Thus we have released only the adequate part of our code and dataset to make the artifact evaluation process clean and convenient- eliminating any manual effort to achieve high performance.

The phase1-mini.csv is only a prototypic representative of phase 1 to ensure the functionality of our released code. Therefore, it is expected to perform subpar. In the coming days, we will release all the phase datasets, as well as, some additional artifacts including preprocessing modules and raw datasets.

> should I sample 2 more small subsets of data from the predicted set, then validate and rectify the labels manually for training?

: Yes, while the actual datasets are being released, this is definitely a solution. Please let us know if you have any questions.

Thanks.

**Comment by Czwbna** [comment]
@CellularLint Thank you for your reply.
